client/Makefile

```diff
@@ -17,7 +17,7 @@ include $(ROOT_DIR_PATH)/dev/makefiles/show-prompt.mk
 
 
 # All available language versions.
-LANGUAGES := go javascript python
+LANGUAGES := jupyterlab go javascript python
```

Comment:
At some point, we might want to move "jupyterlab" to another directory instead of "client". For example "extras" or something like that.